databricks_cli/pipelines/cli.py

```diff
@@ -91,6 +117,7 @@ def delete_cli(api_client, spec_arg, spec, pipeline_id):
     """
     pipeline_id = _get_pipeline_id(spec_arg=spec_arg, spec=spec, pipeline_id=pipeline_id)
     PipelinesApi(api_client).delete(pipeline_id)
+    click.echo("Pipeline {} successfully deleted".format(pipeline_id))
```

Comment:
Nit: Submitting a delete requests deletion of resources, so maybe just say "Pipeline {} deleted". It very rarely fails, and we do retry (even if the service starts up), but probably still best to not say it's successful ;).